CoreLibs/src/org/sleuthkit/autopsy/corelibs/OpenCvLoader.java

```diff
@@ -45,6 +54,7 @@
      *   Note that the stack trace in the exception can be confusing because it refers to a
      *   past invocation. 
      */
+    @Deprecated
```

Comment:
In addition to the deprecated annotation, we also need to add @deprecated to the comment with words top the effect of "Use hasOpenCvLoaded instead."